src/controllers/management-controller.ts

```diff
@@ -108,6 +124,22 @@ export class ManagementController {
     });
   }
 
+  async updateAssetDataAddress(
```

Comment:
please note that with version 0.2.0, there will be the new `/v3/assets` endpoint that will include the `DataAddress` into the `Asset`, so to update it the `update` one will be used.

Reply:
in my opinion, now that we found a workaround for the jsonld library bug, we can go quite fast to 0.2.0 with the new /v3 api so we'll need to implement this once

Reply:
I'm closing this because we're already working on the newer version of the connector that doesn't support PUT on data addresses (separately) anymore. Updating an asset will still be available thanks to #58